src/v/cluster/bootstrap_backend.cc

```diff
@@ -128,6 +131,14 @@ bootstrap_backend::apply(bootstrap_cluster_cmd cmd) {
         }
     }
 
+    // Apply initial node UUID to ID map
+    if (!cmd.value.node_ids_by_uuid.empty()) {
+        co_await _members_manager.invoke_on_all(
```

Comment:
My thinking here was this way is somewhat more future proof in case sevice allocation changes. And also this way is allocation agnostic. But maybe that doesn't make sense as the allocation is unlikely to change.